designsafe/static/scripts/data-depot/components/data-depot-listing/data-depot-listing.component.js

```diff
@@ -148,6 +150,27 @@ class FilesListingCtrl {
             default:
                 return 'fa-file-o';
         }
+    }
+    isNotebook(file) {
+        const extension = file.name.split('.').pop();
+        return extension === 'ipynb';
+    }
+    hasNotebooks() {
+        const listing = this.FileListingService.listings.main.listing;
```

Comment:
`this.listing` inside the controller will give you the current listing without having to call out to the service.